src/styling/articleWriter.css

```diff
@@ -1,33 +1,41 @@
 label {
     padding-right: 50px;
 }
-
-textarea#content {
-    width: 400px;
-    height: 200px;
+textarea#title {
+    width: 200%;
 }
-
 textarea#summary {
-    width: 400px;
+    width: 200%;
     height: 100px;
 }
 
-textarea#title {
-    width: 400px;
+textarea#content {
```

Comment:
I think for consistency, it's best if you apply this width to all the fields.

Reply:
https://github.com/Wolfsvale/Aternos-guides/pull/8/commits/98989d2d467ddf1c94f9f78a72c00bf1889d6d02

Reply:
should be fixed